src/content/docs/apm/agents/ruby-agent/troubleshooting/incorrect-data-unicorn.mdx

```diff
@@ -5,20 +5,19 @@ tags:
   - Agents
   - Ruby agent
   - Troubleshooting
-metaDescription: 'Use expectedbehavior/unicorn-worker-killer to add a before_kill callback to manually shutdown the agent'
+metaDescription: 'Troubleshooting for New Relic Ruby APM monitoring for use of Unicorn and metric and custom event metrics are too low.'
 redirects:
-  - /docs/agents/ruby-agent/troubleshooting/incorrect-counts-unicorn
 ---
 
 ## Problem
 
-You're using New Relic and Unicorn together, but the counts for metrics and custom events seem too small.
+You're using New Relic Ruby APM and Unicorn together, but the counts for metrics and custom events seem too small.
 
 ## Solution
 
 Have Unicorn manually call `NewRelic::Agent.shutdown` to run the agent's exit handlers. Some of Unicorn's exit methods prevent the agent from shutting down as expected. The change below runs New Relic's shutdown method before killing the Unicorn process.
 
-This fix relies on: https://github.com/expectedbehavior/unicorn-worker-killer
+This fix relies on [the unicorn-worker-killer gem](https://github.com/expectedbehavior/unicorn-worker-killer). 
```

Comment:
```suggestion
This fix relies on [@expectedbehavior](https://github.com/expectedbehavior)'s fork of [the unicorn-worker-killer gem](https://github.com/expectedbehavior/unicorn-worker-killer). 
```

Reply:
I specify the fork owner because they aren't the standard owner of the gem. Customers may have this gem already installed, but in the standard form.